app/src/main/java/com/gemwallet/android/features/asset/details/views/AssetDetailsScene.kt

```diff
@@ -78,6 +78,7 @@ fun AssetDetailsScene(
     onSwap: (AssetId, AssetId?) -> Unit,
     onTransaction: (txId: String) -> Unit,
     onChart: (AssetId) -> Unit,
+    onChain: (AssetId) -> Unit,
```

Comment:
Bad name:
- you could use it for any asset
- should use AssetIdAction

Reply:
renamed to openNetwork with AssetIdAction